__mocks__/dotenv.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/no-commonjs */
```

Comment:
You can define the env to be nodejs in eslint (or create an algolia/node subconfig too)

Reply:
putting the env doesn't disable `import/no-commonjs`